polaris-icons/.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    ["@shopify/babel-preset/web", {"modules": "auto", "typescript": true}],
```

Comment:
uugh this is an ooold version of `@shopify/babel-preset`.

It's not a blocker for now, but in a follow up it would be good to get all the babel dependencies set up at the top level of the repo and have us use the latest version of each, in a root level babel.config.js, to avoid having multiple different versions spread across the packages.